spec/ai_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'ai'
+describe Ai do
+  describe "#make_guess" do
+    it "returns a string with 4 letters" do
+      computer = Ai.new
+      
+      expect(computer.make_guess).to eq("RRRR")
```

Comment:
This doesn't assert exactly what the spec says it does. This test is going to break when you actually begin to implement the AI. Maybe expect(computer.make_guess.length).to be(4) would be more resilient to change.